frontend/src/common/hooks/useCategoryFilter.ts

```diff
@@ -166,6 +168,29 @@ function addEmptyCategoryValues(
   }
 }
 
+/**
```

Comment:
I added this function as a way to swap out the filter type per column. I initially anticipated using a `between` filter type for filter by publication date but ended up using the existing `includesSome`. I have kept the functionality as-is (that is, always using `includesSome`) but I expect we will need it - with modifications - for future filters such as cell count, gene count and possible development stage.